Handler/PdfHandler.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\SubmissionBundle\Handler;
+
+use EMS\CommonBundle\Service\Pdf\PdfGenerator;
+use EMS\FormBundle\FormConfig\FormConfig;
+use EMS\FormBundle\Handler\AbstractHandler;
+use EMS\FormBundle\FormConfig\SubmissionConfig;
+use EMS\FormBundle\Submit\AbstractResponse;
+use EMS\FormBundle\Submit\FailedResponse;
+use EMS\SubmissionBundle\FormConfig\PdfConfig;
+use EMS\SubmissionBundle\Service\DirectoryManager;
+use EMS\SubmissionBundle\Service\SubmissionRenderer;
+use EMS\SubmissionBundle\Submit\PdfResponse;
+use Symfony\Component\Form\FormInterface;
+
+final class PdfHandler extends AbstractHandler
```

Comment:
```suggestion
final class PdfCreateHandler extends AbstractHandler
```

Reply:
A second handler is needed for the use case "save on sftp server" --> SftpStorageHandler

Reply:
SftpStorageHandler requires that previousResponse is not null, and implements a new interface to indicate that we have a "saveable" file.